docs/docs/guides/labs/dg/incrementally-adopting-dg/migrating-project.md

```diff
@@ -13,69 +13,93 @@ This guide is only relevant if you are starting from an _existing_ Dagster proje
 
 :::
 
-Let's begin with a Dagster project that has some assets:
+Let's begin with a basic existing Dagster project. Our current working
+directory is the project root. Our project defines a Python
+package with a `setup.py` and a single Dagster asset. The asset is exposed in
+a top-level `Definitions` object in `my_existing_project/definitions.py`.
 
 <CliInvocationExample path="docs_snippets/docs_snippets/guides/dg/migrating-project/1-tree.txt"  />
 
 ## Install dependencies
 
 ### Install the `dg` command line tool
 
-We'll need to install the `dg` command line tool, which is used to scaffold components. We recommend installing `dg` globally using the [`uv`](https://docs.astral.sh/uv/getting-started/installation/) package manager; it can also be installed using `pip`.
+We'll need to install the `dg` command line tool. We recommend installing `dg` globally using the [`uv`](https://docs.astral.sh/uv/getting-started/installation/) package manager; it can also be installed using `pip`.
```

Comment:
Just say to install it using your existing package manager. The point is to show our agnosticism vis-a-vis package managers.